### PR TITLE
Update to ccf-3.0.0-rc1

### LINKED ISSUE #137: Update to ccf-3.0.0-rc0
https://github.com/microsoft/CCF/releases/tag/ccf-3.0.0-rc0

### PR DESCRIPTION
**Pull request by jeffa5**
Fixes #137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ make install-ccf
 Or
 
 ```bash
-$ wget https://github.com/microsoft/CCF/releases/download/ccf-3.0.0-dev6/ccf_3.0.0_dev6_amd64.deb
-$ sudo dpkg -i ccf_3.0.0_dev6_amd64.deb # Installs CCF under /opt/ccf
+$ wget https://github.com/microsoft/CCF/releases/download/ccf-3.0.0-rc0/ccf_3.0.0_rc0_amd64.deb
+$ sudo dpkg -i ccf_3.0.0_rc0_amd64.deb # Installs CCF under /opt/ccf
```
Comment (jumaffre) [inline README.md:23]:
```suggestion
$ wget https://github.com/microsoft/CCF/releases/download/ccf-3.0.0-rc0/ccf_3.0.0_rc0_amd64.deb
$ sudo dpkg -i ccf_sgx_3.0.0_rc0_amd64.deb # Installs CCF under /opt/ccf
```

File: Dockerfile.virtual [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Build
-FROM mcr.microsoft.com/ccf/app/dev:3.0.0-dev6-sgx as builder
+FROM mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-sgx as builder
```
Comment (jumaffre) [inline Dockerfile.virtual:2]:
```suggestion
FROM mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-virtual as builder
```

New containers for this release too.

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   benchmark:
     runs-on: ubuntu-20.04
-    container: mcr.microsoft.com/ccf/app/dev:3.0.0-dev6-sgx
+    container: mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-sgx
```
Comment (jumaffre) [inline .github/workflows/benchmark.yml:10]:
```suggestion
    container: mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-virtual
```

File: Dockerfile.virtual [hunk 4]
```diff
@@ -1,14 +1,14 @@
 # Build
-FROM mcr.microsoft.com/ccf/app/dev:3.0.0-dev6-sgx as builder
+FROM mcr.microsoft.com/ccf/app/dev:3.0.0-rc0-virtual as builder
 
 COPY CMakeLists.txt /src/
 COPY src /src/src/
 COPY proto /src/proto/
 WORKDIR /build
-RUN CC="/opt/oe_lvi/clang-10" CXX="/opt/oe_lvi/clang++-10" cmake -GNinja -DCOMPILE_TARGETS=virtual /src && ninja
+RUN CC="/opt/oe_lvi/clang-10" CXX="/opt/oe_lvi/clang++-10" cmake -GNinja -DCOMPILE_TARGET=virtual /src && ninja
```
Comment (jumaffre) [inline Dockerfile.virtual:8]:
```suggestion
RUN CC=$(command -v clang-10) CXX=$(command -v clang++-10) cmake -GNinja -DCOMPILE_TARGET=virtual /src && ninja
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -20,7 +20,7 @@ jobs:
```
Comment (jumaffre) [inline .github/workflows/ci.yml:20]: The line above should be updated to `run: mkdir -p build && cd build && CC=$(command -v clang-10) CXX=$(command -v clang++-10) cmake -GNinja .. && ninja`

**Comment by jeffa5** [comment]
Given that the CCF builds are now different, should we switch to using docker containers more for things rather than having to keep installing ccf-virtual, then ccf-sgx and back?

**Review by jumaffre — APPROVED** [review]

**Pull request merged**